tests/collections/tts/parts/preprocessing/test_features.py

```diff
@@ -12,59 +12,153 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
+import contextlib
+import tempfile
+from pathlib import Path
+
 import numpy as np
 import pytest
+import soundfile as sf
+import torch
 
-from nemo.collections.tts.parts.preprocessing.features import MelSpectrogramFeaturizer, PitchFeaturizer, compute_energy
+from nemo.collections.tts.parts.preprocessing.features import (
+    EnergyFeaturizer,
+    MelSpectrogramFeaturizer,
+    PitchFeaturizer,
+)
 
 
 class TestTTSFeatures:
+    audio_filename = "test.wav"
+    spec_dim = 80
+    hop_len = 100
+    audio_len = 10000
+    sample_rate = 20000
+    spec_len = 1 + (audio_len // hop_len)
+    manifest_entry = {"audio_filepath": audio_filename}
```

Comment:
nit: I am not sure what is the global test style in NeMo, but from my personal opinion, fixtures and/or `setup_class()` `teardown_class()` are commonly used to initialize variables for a test class. See ref: https://docs.pytest.org/en/6.2.x/xunit_setup.html